circle.yml

```diff
@@ -0,0 +1,51 @@
+machine:
+  environment:
+    PATH: /home/ubuntu/miniconda2/bin:$PATH
+
+dependencies:
+  cache_directories:
+    - "~/nilearn_data"
+
+  pre:
+    # Get rid of existing virtualenvs on circle ci as they conflict with conda.
+    # Trick found here:
+    # https://discuss.circleci.com/t/disable-autodetection-of-project-or-application-of-python-venv/235/10
+    - cd && rm -rf ~/.pyenv && rm -rf ~/virtualenvs
+    # We need to remove conflicting texlive packages.
+    - sudo -E apt-get -yq remove texlive-binaries --purge
+    # Installing required packages for `make -C doc check command` to work.
+    - sudo -E apt-get -yq update
+    - sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install dvipng texlive-latex-base texlive-latex-extra
+
+  override:
+    # Moving to nistats directory before performing the installation.
+    - cd ~/nistats
+    - source continuous_integration/install.sh:
+        environment:
+            DISTRIB: "conda"
+            PYTHON_VERSION: "3.5"
+            NUMPY_VERSION: "*"
+            SCIPY_VERSION: "*"
+            SCIKIT_LEARN_VERSION: "*"
+            MATPLOTLIB_VERSION: "*"
+            PANDAS_VERSION: "*"
+    - conda install sphinx coverage pillow -y -n testenv
+
+    # Generating html documentation (with warnings as errors)
+    # we need to do this here so the datasets will be cached
+    - source continuous_integration/circle_ci_test_doc.sh:
+        timeout: 1500
+
+test:
+  override:
+    - echo "ignore unit tests in circleCI"
+    # - make clean test test-coverage
+    # workaround - make html returns 0 even if examples fail to build
+    # (see https://github.com/sphinx-gallery/sphinx-gallery/issues/45)
+    - cat ~/log.txt && if grep -q "Traceback (most recent call last):" ~/log.txt; then false; else true; fi
```

Comment:
You don't need this line anymore if you are using a recent enough version of sphinx-gallery.